homeassistant/components/swiss_public_transport/__init__.py

```diff
@@ -32,18 +33,19 @@ async def async_setup_entry(
     start = config[CONF_START]
     destination = config[CONF_DESTINATION]
 
+    unique_id = unique_id_from_config(config)
     session = async_get_clientsession(hass)
-    opendata = OpendataTransport(start, destination, session)
+    opendata = OpendataTransport(start, destination, session, via=config.get(CONF_VIA))
 
     try:
         await opendata.async_get_data()
     except OpendataTransportConnectionError as e:
         raise ConfigEntryNotReady(
-            f"Timeout while connecting for entry '{start} {destination}'"
+            f"Timeout while connecting for entry '{unique_id}'"
```

Comment:
Maybe use the config entry title instead of a unique id which is not as readable.
Also use translations

Reply:
Will do, did not know those errors could also be translated :)